src/xdsl/ir.py

```diff
@@ -192,6 +204,16 @@ def create(cls: typing.Type[OperationType],
         return Operation.with_result_types(cls, operands, result_types,
                                            attributes, successors, regions)
 
+    @classmethod
+    def build(cls: typing.Type[T],
+              operands=[],
+              result_types=[],
+              attributes=[],
+              successors=[],
+              regions=[]) -> T:
+        """Create a new operation using builders."""
+        ...
+
     def add_region(self, region: Region) -> None:
         self.regions.append(region)
         region.parent = self
```

Comment:
(I guess I cannot comment on lines that are too far away from the changed ones)
Maybe add a build to block aswell? from_op_list or from_op_list_and_args.

Reply:
This should be done now!